### ISSUE TITLE
Spring Boot 2 and @CircuitBreaker with name not loading config

### ISSUE THREAD

**Issue by jpayne97** [resilience4j/resilience4j#256]
Based on: https://github.com/RobWin/resilience4j-spring-boot-demo#spring-aop

When I add the annotation to either a class or method, I would expect the circuit breaker to be applied per the documentation.  Spring Boot 2 and webflux.  It only works if I set it up from the registry and properties.  Does the annotation only support the equivalent of CircuitBreaker.ofDefaults("name") ?
```
this.circuitBreaker = circuitBreakerRegistry
        .circuitBreaker(
            "exampleBackend",
            () -> circuitBreakerProperties.createCircuitBreakerConfig("exampleBackend")
        );
```

**Comment by RobWin** [comment]
Hi,

the Spring Aspect creates a CircuitBreaker instance and checks if you have configured the CircuitBreaker in your properties/yaml file.
See : https://github.com/resilience4j/resilience4j/blob/master/resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerAspect.java#L68

**Comment by jpayne97** [comment]
I have it setup in yaml file.  Does it not support adding it to Mono or Flux yet?

**Comment by storozhukBM** [comment]
@jpayne97 I think your question is related to #258

**Comment by RobWin** [comment]
Closed in favor of #258 
Will be part of 0.14.0
